libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef-bl.dat

```diff
@@ -0,0 +1,106 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the ZeroOneX6 hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# board ID for firmware load
+APJ_BOARD_ID 5600
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+FLASH_BOOTLOADER_LOAD_KB 128
+
+# flash size
+FLASH_SIZE_KB 2048
+
+env OPTIMIZE -Os
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 UART7 UART5 USART3
+
+# default to all pins low to avoid ESD issues
+DEFAULTGPIO OUTPUT LOW PULLDOWN
```

Comment:
This needs to come before the MCU line to have an effect

Reply:
I've reviewed other hardware versions where this is placed in the same position. Does this still need to be changed, or should it be moved to the third line?

Reply:
it need to be moved, we should correct the other hwdef using this

Reply:
Alright, it has been moved in the hwdef and hwdef-bl file.

Reply:
@andyp1per  Hi, it is still necessary to move this to the original position in the hwdef-bl file , after multiple experiments have shown that  put before the MCU line results  the telem3 being unable to communicate. However, when placed back in the original location,  the telem3 is able to communicate.

Reply:
this line should be removed entirely in the bootloader.....it was a mistake to use it in the bootloader and its present position makes it nonfunctional anyway